docs/general/builtins/scalar-functions.rst

```diff
@@ -1650,6 +1650,34 @@ Example::
     SELECT 1 row in set (... sec)
 
 
+
+.. _scalar-area:
+
+``area(geo_shape)``
+----------------------
+
+Returns: ``double precision``
+
+The ``area`` function calculates the area of the input shape, in square-degrees.
+The input has to be a column of type ``geo_shape``, a valid WKT string or `GeoJSON <https://geojson.org/>`_.
+See :ref:`geo_shape_data_type` for more information.
+
+Below you can find an example that retrieves the shape with the largest area.
+
+Example::
+
+    cr> select
+    ...     shapeName,
+    ...     area(shape) as "area"
+    ... from sys.shapes
```

Comment:
CrateDB does not expose a `sys.shapes` table. The `sys` schema is preserved for internal system tables only.
Our documentation can be tested using `./gradlew itest`, running it will highlight a test failure related to that.

Reply:
Thanks for the comment. I noticed the sys table was also used in geohash, lat, lon functions and removed it from those places also.

Reply:
@ed50245 Thank you for your changes. Unfortunately the examples are still not passing our doctest triggered by `./gradlew itest`.
For the new example related to the `area` function, a table must be created first, see https://github.com/crate/crate/blob/master/docs/general/builtins/scalar-functions.rst?plain=1#L2910 for an example how to do this "hidden".
Otherwise using a literal instead of a column for this example would eliminate the need for a table upfront.

The existing examples of the geohash etc functions were valid as CrateDB exposes a `sys.summit` table and thus there is no need to change them.

Reply:
ok, my mistake then. I have some local issues when running the gradle task due to jar hell. I will make some time to investigate this and correct the pull request. thanks

Reply:
Cleaning the build fragments by running `./gradlew clean` solves jar hell issues most of the time. Hope this helps, otherwise don't hesitate to ask.

Reply:
ok, done